linkcheck/HtmlParser/__init__.py

```diff
@@ -193,7 +194,7 @@ def _resolve_entity (mo):
         # unknown entity -> ignore
         return u''
     try:
-        return unichr(num)
+        return chr(num)
```

Comment:
(but I only accept this because python 2 dies Real Soon Now and the "but chr() returns bytes on 2... oh, there's a hidden `from builtins` import elsewhere that changes everything" gotcha will die with it.)

Reply:
we'll catch this in the py2 cleanup, will we?

Reply:
`except NameError`, so yes